api/src/main/java/org/openmrs/module/fhir/api/util/FHIRConditionUtil.java

```diff
@@ -65,11 +81,237 @@
 			conceptDt.setCoding(dts);
 			fhirCondition.setCode(conceptDt);
 		}
+		fhirCondition.setClinicalStatus(
+				MapOpenmrsStatusToFHIRClinicalStatus(condition));
 
 		return fhirCondition;
 	}
 
-	public static org.openmrs.Condition generateOpenMRSCondition() {
-		return null;
+	/**
+	 * generates OpenmrsCondition from FHIR Conditions
```

Comment:
Can we start the sentence with a capital letter?

Reply:
Does it generate it from one condition or a list of conditions?

Reply:
one condition